auto_round/special_model_handler.py

```diff
@@ -12,11 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from .utils import logger, torch
+import torch
 
 share_attention_mask_tuple = ("baichuan",)
 special_states_dim_tuple = ("chatglm",)
-not_share_position_ids_tuple = ("llava","phi3_v")
+not_share_position_ids_tuple = ("llava", "phi3_v",)
+Vison_blocks_tuple = ("vision", "visual",)
+MoE_blocks_tuple = ("moe", "expert",)
```

Comment:
do not handle moe as sepcial model . there are many moe models, which not named block as expert.